scripts/hal_alignment/hal_to_maf.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Extract a MAF alignment from a HAL file.
+
+Examples::
+    # Extract a MAF alignment with chromosome 19 of GRCh38 as reference.
+    python hal_to_maf.py --ref-genome GRCh38 --ref-sequence chr19 \
+      --keep-genomes-file genome_list.txt input.hal output.maf
+
+"""
+
+from argparse import ArgumentParser
+import subprocess
+
+from Bio.AlignIO.MafIO import MafIterator, MafWriter
+
+
+if __name__ == '__main__':
+
+    parser = ArgumentParser(description=__doc__)
+    parser.add_argument('hal_file', help="Input HAL file.")
+    parser.add_argument('maf_file', help="Output MAF file.")
+
+    parser.add_argument('--ref-genome', metavar='STR', required=True,
+                        help="Reference genome for output MAF file.")
+    parser.add_argument('--ref-sequence', metavar='STR', required=True,
+                        help="Output sequence within reference genome.")
+    parser.add_argument('--keep-genomes-file', metavar='PATH', required=True,
```

Comment:
In this case the argument has a different type of input, so I find it appropriate to leave the change. However, I leave to @twalsh-ebi to decide if adding the alternative argument `--target-genomes` would be useful or not.

Reply:
I would actually advocate keeping this as `--keep-genomes-file`, partly because as @JAlvarezJarreta mentions it has a different type of input to `--targetGenomes`, but also because there is a possibility that some extra processing will be done to the argument value in a future version of this script, causing it to diverge further from `--targetGenomes`.